payments-core/src/main/java/com/stripe/android/networking/PaymentAnalyticsRequestFactory.kt

```diff
@@ -199,42 +197,30 @@ class PaymentAnalyticsRequestFactory @VisibleForTesting internal constructor(
         tokenType: Token.Type? = null,
         threeDS2UiType: ThreeDS2UiType? = null
     ): AnalyticsRequest {
-        return AnalyticsRequest(
-            createParams(
-                event.toString(),
-                productUsageTokens,
-                sourceType,
-                tokenType,
-                threeDS2UiType
-            ),
-            RequestHeadersFactory.Analytics.create()
+        return createRequest(
+            event,
+            additionalParams(
+                productUsageTokens = productUsageTokens,
+                sourceType = sourceType,
+                tokenType = tokenType,
+                threeDS2UiType = threeDS2UiType
+            )
         )
     }
 
-    private fun createParams(
-        event: String,
+    private fun additionalParams(
         productUsageTokens: Set<String> = emptySet(),
         @Source.SourceType sourceType: String? = null,
         tokenType: Token.Type? = null,
         threeDS2UiType: ThreeDS2UiType? = null
     ): Map<String, Any> {
-        return createStandardParams(event)
-            .plus(createAppDataParams())
-            .plus(
-                defaultProductUsageTokens.plus(productUsageTokens)
-                    .takeUnless { it.isEmpty() }?.let {
-                        mapOf(FIELD_PRODUCT_USAGE to it.toList())
-                    }.orEmpty()
-            )
+        return defaultProductUsageTokens
+            .plus(productUsageTokens)
+            .takeUnless { it.isEmpty() }?.let { mapOf(FIELD_PRODUCT_USAGE to it.toList()) }
+            .orEmpty()
             .plus(sourceType?.let { mapOf(FIELD_SOURCE_TYPE to it) }.orEmpty())
             .plus(createTokenTypeParam(sourceType, tokenType))
-            .plus(
-                threeDS2UiType?.let {
-                    mapOf(
-                        FIELD_3DS2_UI_TYPE to it.toString()
-                    )
-                }.orEmpty()
-            )
+            .plus(threeDS2UiType?.let { mapOf(FIELD_3DS2_UI_TYPE to it.toString()) }.orEmpty())
```

Comment:
Just format changes here + removed `createStandardParams(event).plus(createAppDataParams())` that now live on the parent AnalyticsFactory.